client/src/components/cluster/RecommendedCategories/RecommendedCategories.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import CategoryContainer from './CategoryContainer.js';
+import { listOne, listTwo } from './mockData.js';
+
+class RecommendedCategories extends Component {
+    render() {
+        const makeRow = item => (
+            <CategoryContainer key={item.id}
+                category={item.category}
+                url={item.url}
+            />
+        );
+
+        const lists = [listOne, listTwo];
+
+        return (
+            <div className="category-section">
+                <div className="section-header">
+                    <h2>Didn't find what you're looking for?</h2>
+                    <h1>Browse recommended categories</h1>
```

Comment:
Maybe make this tag h3 since it's below the h2?

Reply:
In the mock the "Browse Recommended Categories" is bigger than the "Didn't find what you're looking for?"

Reply:
Hmm. Maybe change this to a link instead? Just link it to any link for the time being

Using h1 is a bit misleading since there should only be 1 H1 on the page and the headers reflect how the page is laid out

https://www.w3.org/WAI/tutorials/page-structure/headings/